packages/components/src/internal/components/editable/actions.ts

```diff
@@ -699,12 +709,19 @@ export async function updateGridFromBulkForm(
     dataRowIndexes: List<number>,
     lockedOrReadonlyRows: number[],
     isIncludedColumn: (col: QueryColumn) => boolean,
-    containerPath: string
+    containerPath: string,
+    useAdditionalCols: boolean = false
```

Comment:
Since `useAdditionalCols` is always using the cols from the `editorModel` I think we should name it `useEditorModelCols`.